### PR TITLE
Create Cqc.org.uk.xml

### PR DESCRIPTION
**Pull request by sampablokuper**

### CODE REVIEW

File: src/chrome/content/rules/Cqc.org.uk.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="cqc.org.uk">
+	<target host="cqc.org.uk" />
+	<target host="www.cqc.org.uk" />
```
Comment (J0WI) [inline src/chrome/content/rules/Cqc.org.uk.xml]: See also:
```
webdataforms.cqc.org.uk
services.cqc.org.uk
jobs.cqc.org.uk
communities.cqc.org.uk

**Comment by jeremyn** [comment]
@sampablokuper Are you still interested in working on this pull request?

**Comment by sampablokuper** [comment]
@jeremyn , yes in principle, but I'm afraid I can't spare time at the moment. Anyone interested should feel free to work on it.

**Comment by J0WI** [comment]
I've added the missing hosts.

**Comment by jeremyn** [comment]
- [ ] Add a `target` for https://qrp.cqc.org.uk

**Comment by J0WI** [comment]
Added.

**Pull request merged**